### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by SergeySheps**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="styles/footer.css">
+    <link rel="stylesheet" href="styles/media.css">
+    
+    <title>Document</title>
```
Comment (SergeySheps) [inline index.html]: Please add title "PSD task"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+                        <li>About</li>
+                        <li>Services</li>
+                        <li>Folio</li>
+                        <li>Contact</li>
+                    </ul>
+                </nav>
+            </div>
+        </menu>
+        <div class="header__slider">
+            <img src="images/arrow-left.png" alt="" class="header__slider_leftArrow">
```
Comment (yankovichdmitry) [inline index.html]: It's better to use this as links, because it's not semantic img

File: index.html [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+            <img src="images/slider_header_1.jpg" alt="" class="header__slider-image ">
```
Comment (yankovichdmitry) [inline index.html]: Same as above, it's background image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+            <div class="header__slider-container">
+                <img src="images/perosn.png" alt="" class="header__slider_image-person header__slider-container-item">
+                <div class="header__slider-mainText header__slider-container-item">Nunc digno lorem dol</div>
+                <div class="header__slider-sideText header__slider-container-item">Vivamus vestibulum nulla</div>
+                <button class="header__slider-button header__slider-container-item">Loareet ipon</button>
+            </div>
+            <img src="images/arrow-right.png" alt="" class="header__slider_rightArrow">
```
Comment (yankovichdmitry) [inline index.html]: Same as above

File: index.html [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+                        <li>Home</li>
+                        <li>About</li>
+                        <li>Services</li>
+                        <li>Folio</li>
+                        <li>Contact</li>
+                    </ul>
+                </nav>
+            </div>
+        </menu>
+        <div class="header__slider">
```
Comment (yankovichdmitry) [inline index.html]: I think that this content should not be included into header

File: index.html [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+                <div class="header__slider-mainText header__slider-container-item">Nunc digno lorem dol</div>
+                <div class="header__slider-sideText header__slider-container-item">Vivamus vestibulum nulla</div>
+                <button class="header__slider-button header__slider-container-item">Loareet ipon</button>
+            </div>
+            <img src="images/arrow-right.png" alt="" class="header__slider_rightArrow">
+        </div>
+    </header>
+    <main>
+        <div class="features">
+            <div class="features_item features_item-1">
```
Comment (yankovichdmitry) [inline index.html]: I don't like the usage of `features_item-1` like classes. If these elements are unique, you should use ids.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+                    <div class="shadow shadow-last"></div>
+                <div class="features__container">
+                    <img src="images/dialog.png" alt="" class="features_image">
+                    <div class="features__container_mainText">egestasmod</div>
+                    <div class="features__container_sideText">Felis moodo tortor</div>
+                </div>
+            </div>
+        </div>
+        <div class="main__content">
+            <div class="main__content_item">
```
Comment (yankovichdmitry) [inline index.html]: I don't think, that it's modificator.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+                    <img src="images/444.png" alt="" class="newsCard__image">
+                    <div class="newsCard__article">
+                        Pellentesque odio nisi, euismod in, pharetra a, ultricies in.
+                    </div>
+                    <button class="newsCard__button">
+                        Read More
+                    </button>
+                </div>
+            </div>
+            <div class="main__content_item main__content_item-last">
```
Comment (yankovichdmitry) [inline index.html]: and here you could create modificator

File: index.html [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+            </div>
+        </div>
+    </main>
+    <footer>
+        <div class="footer__content">
+            <div class="footer__topPart">
+                <div class="footer__topPart_item">Copyright (c) 2013 
+                    <div class="footer__topPart_item_text">All rights reserved.</div></div>
+                <div class="footer__topPart_item">Designed by:
+                    <div class="footer__topPart_item_text">www.alltemplateneeds.com</div></div>
```
Comment (yankovichdmitry) [inline index.html]: Should be link

File: index.html [hunk 10]
```diff
@@ -0,0 +1,288 @@
...
+                <div class="footer__topPart_item">Imags from:
+                    <div class="footer__topPart_item_text">wallcoo.net | wallpaperswide.com</div>
```
Comment (yankovichdmitry) [inline index.html]: Same as above

File: index.html [hunk 11]
```diff
@@ -0,0 +1,288 @@
...
+                </div>
+                <div class="footer__topPart_item">Contact
+                    <div> <a href="/">info@companyname.com</a></div>
```
Comment (yankovichdmitry) [inline index.html]: use `href="mailto:"`

File: index.html [hunk 12]
```diff
@@ -0,0 +1,288 @@
...
+                </div>
+            </div>
+            <div class="footer__bottomPart">
+                <ul class="footer__bottomPart_item footer__bottomPart_socNetWork">
+                    <li><img src="images/35.png" alt=""></li>
```
Comment (yankovichdmitry) [inline index.html]: not semantic images, use links

File: index.html [hunk 13]
```diff
@@ -0,0 +1,288 @@
...
+                    <li><img src="images/34.png" alt=""></li>
+                    <li><img src="images/33.png" alt=""></li>
+                    <li><img src="images/32.png" alt=""></li>
+                    <li><img src="images/31.png" alt=""></li>
+                </ul>
+                <div class="footer__bottomPart_item">
+                    Home :: About :: Contact
```
Comment (yankovichdmitry) [inline index.html]: Use links please

File: index.html (hunk 12 above)
Comment (SergeySheps) [reply index.html]: qweqw

File: styles/banner.css [hunk 14]
```diff
@@ -0,0 +1,183 @@
...
+  box-sizing: border-box;
+  flex-flow: row wrap;
+  max-width: 1170px;
+  margin: auto;
+  background: #ffffff;
+  box-shadow: 7px 10px 44px -31px rgba(0, 0, 0, 0.75);
+  border-radius: 0px 0px 5px 5px;
+  margin-bottom: 55px;
+}
+.features_item {
```
Comment (yankovichdmitry) [inline styles/banner.css]: If it's element from block `features` then name should be `features__item`

File: styles/banner.css [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+  margin: 30px 0px 35px 0px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column nowrap;
+  text-align: center;
+  width: 100%;
+  max-width: 130px;
+}
+.features__container_mainText {
```
Comment (yankovichdmitry) [inline styles/banner.css]: You should use element, not modificator

File: styles/banner.css [hunk 16]
```diff
@@ -0,0 +1,183 @@
...
+  color: rgba(255, 255, 255, 0.6);
+  text-transform: uppercase;
+  line-height: 1.4;
+  text-align: center;
+  font-size: 17px;
+  color: #747474;
+  margin-top: 20px;
+  margin-bottom: 2px;
+}
+.features__container_sideText {
```
Comment (yankovichdmitry) [inline styles/banner.css]: Same as above

File: styles/footer.css [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+  display: flex;
+  font-weight: bold;
+  align-items: center;
+  justify-content: flex-start;
+}
+.footer__bottomPart-item {
+  color: #b297c4;
+  margin: 40px 160px 65px 0;
+}
+.footer__bottomPart a {
```
Comment (yankovichdmitry) [inline styles/footer.css]: You should not create css rules for selectors in bem. Please, add element class for this selectors

File: styles/footer.css [hunk 18]
```diff
@@ -0,0 +1,89 @@
...
+  color: #b297c4;
+}
+.footer__bottomPart-socNetWork {
+  list-style: none;
+  padding: 0;
+}
+.footer__bottomPart-socNetWork li {
```
Comment (yankovichdmitry) [inline styles/footer.css]: Same as above

File: styles/media.css [hunk 19]
```diff
@@ -0,0 +1,164 @@
...
+  }
+  .banner__slider-button {
+    font-size: 12px;
+  }
+  .banner__slider_image-person {
+    margin-top: 87px;
+    height: 60px;
+    width: 53.33333333px;
+  }
+  .menu__rightside_MainMenu li {
```
Comment (yankovichdmitry) [inline styles/media.css]: `MainMenu` is not modificator

File: styles/banner.less [hunk 20]
```diff
@@ -0,0 +1,124 @@
+@import "mixins.less";
+@shadow-width: 9px; 
+.banner__slider{
+    max-width: @width_viewScreen;
+    margin: auto;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    &_leftArrow{
+        .arrow();
```
Comment (yankovichdmitry) [inline styles/banner.less]: Maybe you could create an arrow as an element, and left/right arrows as modificators? You could even have 1 background image, and use `transform: rotate(180deg);`

File: styles/banner.less [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+        justify-content: center;
+        display: flex;
+        border-left: 20px solid white;
+        position: relative;
+        flex: 150px 1 1;
+    }
+
+}
+
+.features__container{
```
Comment (yankovichdmitry) [inline styles/banner.less]: could be included into `.features` with &__container

File: styles/banner.less [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+    background-image: url("../images/shadow.png");
+    background-repeat: no-repeat;
+    position: absolute;
+    // left: calc(100% - @shadow-width);
+    left: calc(100% + 20px);
+    width: @shadow-width;
+    background-size: 100% 100%;
+    height: 100%;
+}
+.shadow_last{
```
Comment (yankovichdmitry) [inline styles/banner.less]: include into `.shadow` as &_last

**Pull request closed without merging**